pkgs/development/tools/rust/cargo-make/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchFromGitHub, rustPlatform }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "cargo-make";
+  version = "0.19.1";
+
+  src = fetchFromGitHub {
+    owner = "sagiegurari";
+    repo = pname;
+    rev = version;
+    sha256 = "1bv5gv5zd9h0yw7wb28r0c4z9wqa6n7ac4ipd855apwz1v7sm9n3";
+  };
+
+  cargoSha256 = "1l7fg1fq76g4bsyfffx338fcxvxb50qdwys8dww774k80dmdw4nh";
+  cargoPatches = [ ./0001-cargo-lock.patch ];
```

Comment:
Can you upload this lockfile in a gist? we try to avoid growing the size of nixpkgs.

Reply:
Thanks for the review.
I have uploaded the `Cargo.lock` in a gist as your suggestion.
Please review it again, if possible.